preludezdev/NaverMovieSearchApi2/app/src/main/java/com/example/navermoviesearchapi2/di/RepositoryModule.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.navermoviesearchapi2.di
+
+import com.example.navermoviesearchapi2.data.MovieRepository
+import com.example.navermoviesearchapi2.data.source.MovieRepositoryImpl
+import com.example.navermoviesearchapi2.data.source.remote.MovieRemoteDataSource
+import com.example.navermoviesearchapi2.data.source.remote.MovieRemoteDataSourceImpl
+import org.koin.core.qualifier.named
+import org.koin.dsl.module
+
+val repositoryModule = module {
+    single { MovieRepositoryImpl(get(named("Remote"))) as MovieRepository }
+    single(named("Remote")) { MovieRemoteDataSourceImpl(get()) as MovieRemoteDataSource }
```

Comment:
```suggestion
    single<MovieRemoteDataSource> { MovieRemoteDataSourceImpl(get()) }
```
이렇게 하면 name을 짓지 않아도 되지않을까요 ??

Reply:
수정했습니다
ccb9b4a